tests/pipeline/nodes/model/hrnetv1/test_hrnet.py

```diff
@@ -0,0 +1,71 @@
+"""
+Copyright 2021 AI Singapore
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import os
+import yaml
+import pytest
+import cv2
+import numpy as np
+import numpy.testing as npt
+
+from peekingduck.pipeline.nodes.model.hrnet import Node
+
+
+@pytest.fixture
+def hrnet_config():
+    filepath = os.path.join(os.getcwd(), 'tests/pipeline/nodes/model/hrnetv1/test_hrnet.yml')
+    with open(filepath) as file:
+        node_config = yaml.safe_load(file)
+    node_config['root'] = os.getcwd()
+    return node_config
+
+
+@pytest.fixture
+def hrnet(request, hrnet_config):
+    node = Node(hrnet_config)
+
+    return node
+
+
+class TestHrnet:
+
```

Comment:
To speed up unittesting, we will mark model testing and run those non model testing when merging. 

```suggestion
@pytest.mark.mlmodel
class TestHrnet:

```

Reply:
okay, i marked the unit test for detector as well since it has the model initialization portion